README.md

```diff
@@ -2,92 +2,91 @@
 
 [![codecov](https://codecov.io/gh/Michigan-Tech-Courses/rate-my-professors/branch/master/graph/badge.svg?token=YSBV5T5GVY)](https://codecov.io/gh/Michigan-Tech-Courses/rate-my-professors)
 
-A basic wrapper for Rate My Professors's GraphQL API. Includes TypeScript definitions.
+A basic wrapper for Rate My Professors' GraphQL API, including TypeScript definitions. This package retrieves school and professor information with capabilities to retrieve detailed ratings of professors.
 
-It is possible to pull full ratings with content as well, but currently this package just returns the average.
+## 🚀 Getting Started
 
-## 🏗 Usage
+### Installation
+To install the package, run:
 
-```js
-// Change to 
+```bash
+npm install @mtucourses/rate-my-professors
+or
+yarn add @mtucourses/rate-my-professors // recommended
```

Comment:
I personally wouldn't explictly recommend Yarn as v1 has reached its [end of life](https://github.com/yarnpkg/yarn/issues/8583#issuecomment-784017423)

Reply:
Alright, we can delete that part. No worries.